packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2236,6 +2236,7 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
   }
 
   void _didChangeTextEditingValue() {
+    _formatAndSetValue(widget.controller.value);
```

Comment:
The addition of this line is causing a lot of tests to fail in editable_text_test.dart.  I think this line will cause a bunch of unnecessary edits to the value.

Reply:
Just noticed this now. I will find a way to just trigger the formatter method without calling it on every change.

Reply:
Actually any change I do to call the validator method it breaks some test. I fear this will be a pontential breaking change.

Reply:
@justinmc back to this.

Do you have anhy suggestion to work around the problems I mentioned above? I may give another try soon.